shell/pom.xml

```diff
@@ -89,6 +89,12 @@
       <artifactId>alluxio-job-client</artifactId>
       <version>${project.version}</version>
     </dependency>
+    <dependency>
```

Comment:
this dependency will be avoided by the registry? this dependency will be an issue when invoked using the rest server

Reply:
please use the pattern in this PR https://github.com/Alluxio/alluxio/pull/11586 to re-use `ValidationToolFactory`